nbbuild/javadoctools/template.xml

```diff
@@ -308,10 +353,14 @@ cause it to fail.
             <bottom>${javadoc.footer}</bottom>
             <!-- Avoid timestamp comments in _all_ html generated files, to reduce unnecessary git commits -->
             <arg value="-notimestamp" />
-            <arg value="--allow-script-in-comments" />
         </javadoc>
     </target>
-    
+
+    <target name="javadoc-exec-packages" depends="javadoc-init,javadoc-generate-references,javadoc-generate-overview,javadoc-exec-condition,javadoc-check-timestamps,javadoc-make-plain-title,javadoc-make-hyperlinked-title,javadoc-exec-condition,-javadoc-set-footer,-javadoc-exec-packages-jdk,-javadoc-exec-packages-nbjavac" unless="javadoc.exec.packages">
+        <!-- javadoc-exec-packages-jdk  -->
+        <!-- javadoc-exec-packages-nbjavac  -->
+    </target>
+
```

Comment:
previous target javadoc-exec-packages depends on -javadoc-exec-packages-jdk and javadoc-exec-packages-nbjavac